### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by White2White** [PaddlePaddle/PaddleNLP#5634]
### 软件环境

```Markdown
- paddlepaddle:
- paddlepaddle-gpu: 2.4.2.post112
- paddlenlp:2.5.2
```

### 重复问题

- [X] I have searched the existing issues

### 错误描述

```Markdown
使用nvidia官方镜像：11.2.2-cudnn8-devel-ubuntu20.04

加载模型时，运行正常，在执行text classifier 的multi class infernece的时候报错
ine 175, in forward\n    out = F.linear(\n  File \"/usr/local/lib/python3.8/dist-packages/paddle/nn/functional/common.py\", line 1886, in linear\n    return _C_ops.linear(x, weight, bias)\nOSError: (External) CUBLAS error(15). \n  [Hint: Please search for the error code(15) on website (https://docs.nvidia.com/cuda/cublas/index.html#cublasstatus_t) to get Nvidia's official solution and advice about CUBLAS Error.] (at /paddle/paddle/phi/kernels/funcs/blas/blas_impl.cu.h:35)\n  [operator < linear > error]
```

### 稳定复现步骤 & 代码

...
for batch in dev_data_loader:
    label = batch.pop("labels")
    logits = model(**batch) (该行报错)
    prob = F.softmax(logits, axis=1)
    labels.extend(label.numpy())
    probs.extend(prob.numpy())
...

**Comment by White2White** [comment]
W0412 10:53:25.662729     7 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 11.2, Runtime API Version: 11.7
W0412 10:53:25.669209     7 gpu_resources.cc:91] device: 0, cuDNN Version: 8.1.
